packages/tables/src/elements/Table.tsx

```diff
@@ -10,14 +10,16 @@ import PropTypes from 'prop-types';
 import { StyledTable, IStyledTableProps } from '../styled';
 import { TableContext } from '../utils/useTableContext';
 
+interface ITableProps extends IStyledTableProps, HTMLAttributes<HTMLTableElement> {
+  /** Removes interactive styling from Table Rows */
```

Comment:
Here we go with lowercase, for consistency. So ~Table Rows~ -> "table rows". The exception would be a direct reference to "component X" which we shouldn't see much (or any) of within prop descriptions.

Reply:
Updated to be lowercase.